apps/perf-test/tasks/fluentPerfRegressions.ts

```diff
@@ -0,0 +1,167 @@
+import * as _ from 'lodash';
+import * as path from 'path';
+
+// TODO: remove. temporary relative pathing until perf-test packages are consolidated.
+import config from '../../../scripts/config';
```

Comment:
You should be able to import this from `@uifabric/build` instead of a relative path I think?